lib/include/openamp/rpmsg.h

```diff
@@ -344,7 +344,7 @@ void rpmsg_destroy_ept(struct rpmsg_endpoint *ept);
 static inline unsigned int is_rpmsg_ept_ready(struct rpmsg_endpoint *ept)
 {
 	return (ept->dest_addr != RPMSG_ADDR_ANY) &&
-		(ept->addr != RPMSG_ADDR_ANY);
+		(ept->addr != 0) && (ept->addr != RPMSG_ADDR_ANY);
```

Comment:
endpoint address can be 0.

Reply:
> endpoint address can be 0. It is possible for existing RPMsg user to use OpenAMP implementation on one end, and use non RPMsg implementation on the other end.

Yes, that's why I just check the local adress(not dest_addr) equals zero, since the other end may allocate zero as a normal address by non RPMsg impelementation. 
But with the change at line 30, OpenAMP doesn't allocate zero as the local address anymore.

Reply:
I have  seen Linux kernel reserved some endpoint address for RPMsg services (0 to 1024), but Linux kernel side doesn't how it expects those reserved services endpoint to setup. And thus, I have some concerns on we have some OpenAMP specific reservation due to implementation convenience.

I am fine with this change, want to see other existing users' input.